streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -107,14 +107,19 @@ public void process(Exchange exchange) {
 			final String PASS = "debezium";
 			String sqlStmt = null;
 			try {
-				Class.forName("com.mysql.jdbc.Driver");
+				Class.forName("com.mysql.cj.jdbc.Driver");
 				
 				//Connecting to openmrs database
 				conn = DriverManager.getConnection(DB_URL, USER, PASS);
 				
 				//Creating statement
 				stmt = conn.createStatement();
-				String sql = config.getJoinClause();
+				String parent_tableValue = config.getParentTable();
+				String parentForeignKeyValue = config.getParentForeignKey();
+				String childPrimaryKeyValue = payload.get(config.getChildPrimaryKey()).toString();
+				
+				String sql = "SELECT uuid FROM {parent_tableValue} WHERE {parentForeignKeyValue} = {childPrimaryKeyValue}";
```

Comment:
>"SELECT uuid FROM {parent_tableValue} WHERE {parentForeignKeyValue} = {childPrimaryKeyValue}";
@gitcliff ,i just gave that as a an example,
you need to find a way of passing in the parameters into that string, otherwise that string cant be executed as it is.

you can use the [Java](https://www.w3schools.com/python/ref_string_format.asp) `String.format` to parametrize Strings. or  [JDBC](https://www.codemeright.com/blog/post/named-parameterized-query-java) to parametrize the sql query
or some other way

Reply:
@mozzy11 i just updated the PR because this i just a sample and still failing on the sql execution against the DB with `java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '} WHERE {parentForeignKeyValue} = {childPrimaryKeyValue}' at line 1`

Reply:
exactly , thats what i was saying .you need to parametrize that string to make a meaningful sql query